src/pyfmi/fmi3.pyx

```diff
@@ -1895,6 +1895,10 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
         """
         Extract the names of the variables in a model.
 
+        Usage of multiple parameters acts as logical AND, e.g.,
```

Comment:
Note; we already test this. Thought it was worth to add this clarification, since this might not be 100% obvious.